fsp_email.py

```diff
@@ -99,35 +115,25 @@ def process_message(config, msg):
         raise UnusableEmail("Email does not come from notify@flightschedulepro.com")
     if msg.is_multipart() and msg.get_content_subtype() == 'alternative':
         for part in msg.get_payload():
-            if part.get_content_type() == 'text/html':
+            if part.get_content_type() == 'text/plain':
                 msg = part
-    if msg.get_content_type() != 'text/html':
-        raise UnusableEmail("Email was expected to be in HTML format but is not")
+    if msg.get_content_type() != 'text/plain':
+        raise UnusableEmail("Email was expected to be in plain format but is not")
     try:
         top = BeautifulSoup(msg.get_payload(decode=True))
```

Comment:
This message part is text/plain, not text/html. You shouldn't be using BeautifulSoup, which is an HTML parser, to parse it.